appinterface/projection/crossfire/crossfire_test.go

```diff
@@ -256,12 +258,11 @@ var _ = Describe("Crossfire", func() {
 			_ = pgMigrate.Reset()
 			pgMigrate.MustUp()
 			server = ghttp.NewServer()
-
-			server.AppendHandlers(
-				ghttp.CombineHandlers(
-					ghttp.VerifyRequest("GET", "/"),
-					ghttp.RespondWith(http.StatusOK, `[]`),
-				))
+			server.RouteToHandler(
```

Comment:
`RouteToHandler` could do the trick, and I think currently the fake server could serve our testing need~ @cdc-Hitesh